### ISSUE TITLE
pool function problem

### ISSUE THREAD

**Issue by JeneeeF** [MPolaris/onnx2tflite#30]
hi,
when I try to convert my onnx model which consists of a maxpool layer, I found the maxpool layer operates wrong, due to the "pool size" and "strides" params. Please confirm whether the following codes in Class TFAveragePool() and TFMaxPool() of  "layers/common_layers.py" needs to be modified from:

> self.avg_pool = keras.layers.AveragePooling2D(pool_size=node_attribute.get("kernel_shape", [2])[0], 
                                                        strides=node_attribute.get("strides", [1])[0], padding=pad_mode)
> self.max_pool = keras.layers.MaxPool2D(pool_size=node_attribute.get("kernel_shape", [2])[0], 
                                                 strides=node_attribute.get("strides", [1])[0], padding=pad_mode)
to :

> self.avg_pool = keras.layers.AveragePooling2D(pool_size=node_attribute.get("kernel_shape", [2]), 
                                                        strides=node_attribute.get("strides", [1]), padding=pad_mode)
> self.max_pool = keras.layers.MaxPool2D(pool_size=node_attribute.get("kernel_shape", [2])[0], 
                                                 strides=node_attribute.get("strides", [1]), padding=pad_mode)

**Comment by MPolaris** [comment]
Thanks for your report.
Maybe these operators have two types: int and list, I will check it carefully and fix it in next commit.
